draft-fossati-tls-attestation.md

```diff
@@ -154,74 +154,108 @@ Several formats for encoding evidence are available, such as:
+The following terms are used in this document:
 
-# Conventions and Terminology
+{: vspace="0"}
+
+TLS Identity Key (TIK): : A cryptographic key used by one of the peers to
+authenticate itself during the TLS handshake.
+
+The reader is assumed to be familiar with the vocabulary and concepts defined in
+{{-rats-arch}}, and those in {{-rats-kat}}.
+
+"Remote attestation evidence" is more succintly referred to as "evidence", and
+"remote attestation results" is more succintly referred to as "results"
+throughout this document. "Remote attestation credentials", or "credentials", is
+used to refer to both evidence and results, when no distinction needs to be made
+between them.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
 document are to be interpreted as described in RFC 2119 {{RFC2119}}.
 
 # Overview
 
-The Remote Attestation Procedures (RATS) architecture {{I-D.ietf-rats-architecture}}
-defines two types of interaction models for attestation, namely the passport model
-and the background check model. The subsections below explain the difference in their 
+The Remote Attestation Procedures (RATS) architecture {{RFC9334}} defines two
+types of interaction models for attestation, namely the passport model and the
+background check model. The subsections below explain the difference in their
 interactions.
 
-As typical with new features in TLS, the client indicates support for the new 
-extension in the ClientHello message. The newly introduced extensions allow evidence
-and nonces to be exchanged. The nonces are used for guaranteeing freshness of
-the exchanged evidence.
-
-When the evidence extension is successfully negotiated, the content of the
-Certificate message contains a payload that is encoded based on the wrapper defined 
-in {{I-D.ftbs-rats-msg-wrap}}.
-
-In TLS a client has to demonstrate possession of the private key via the CertificateVerify
-message, when client-based authentication is requested. The attestation payload
-must contain a key attestation token, which associates a private key with the
-attestation information. An example of a key attestation token format utilizing 
-the EAT format can be found in {{I-D.bft-rats-kat}}.
-
-The recipient extracts evidence from the Certificate message and relays it to the 
-verifier to obtain attestation results. Subsequently, the attested key is used
-to verify the CertificateVerify message.
-
-# Use of Evidence with the Background Check Model
-
-The background check model is described in Section 5.2 of 
-{{I-D.ietf-rats-architecture}} and allows the following modes
-of operation when used with TLS, namely:
+As typical with new features in TLS, the client indicates support for the new
+extension in the ClientHello message. The newly introduced extensions allow
+remote attestation credentials and nonces to be exchanged. The nonces are used
```

Comment:
Why does nonce need to be mentioned separately? Is it not a part of the "remote attestation credentials"? For example, KAT includes nonce, right?

Reply:
I believe nonces are mentioned separately because they have to be passed along explicitly using the new extensions (for background check, at least), separate from the credentials.

Reply:
Then probably define the "remote attestation credentials" such that it is clear that nonce is not a part of that.

Reply:
Do you mean in the `Conventions and Terminology` section or in this paragraph? I don't think nonces are usually lumped into "credentials", though perhaps this is confusing since the nonces are incorporated in credentials.